packages/browser/src/lib/client-hints/interfaces.ts

```diff
@@ -0,0 +1,46 @@
+// https://wicg.github.io/ua-client-hints/#dictdef-navigatoruabrandversion
+export interface NavigatorUABrandVersion {
+  readonly brand: string
+  readonly version: string
+}
+
+// https://wicg.github.io/ua-client-hints/#dictdef-uadatavalues
+export interface UADataValues {
+  readonly brands?: NavigatorUABrandVersion[]
+  readonly mobile?: boolean
+  readonly platform?: string
+  readonly architecture?: string
+  readonly bitness?: string
+  readonly model?: string
+  readonly platformVersion?: string
+  /** @deprecated in favour of fullVersionList */
+  readonly uaFullVersion?: string
+  readonly fullVersionList?: NavigatorUABrandVersion[]
+  readonly wow64?: boolean
+}
+
+// https://wicg.github.io/ua-client-hints/#dictdef-ualowentropyjson
+export interface UALowEntropyJSON {
+  readonly brands: NavigatorUABrandVersion[]
+  readonly mobile: boolean
+  readonly platform: string
+}
+
+// https://wicg.github.io/ua-client-hints/#navigatoruadata
+export interface NavigatorUAData extends UALowEntropyJSON {
+  getHighEntropyValues(hints: string[]): Promise<UADataValues>
+  toJSON(): UALowEntropyJSON
+}
+
+/**
+ * Array of high entropy Client Hints to request. These may be rejected by the user agent - only required hints should be requested.
+ * @return {Object} Object containing low entropy hints and successfully resolved high entropy hints
```

Comment:
These jsdoc comments seem obsolete :-)

Reply:
Oh you mentioned wanting a doc string and I thought you meant for this interface 😅 which did you mean?

Reply:
Oh, I see what you did. The docstring goes in the InitOptions interface.